src/data/admin/Mentors.tsx

```diff
@@ -56,6 +75,7 @@ export const DROPDOWN = ({ object }) => {
               toggle={object.availability.includes(text)}
               text={text}
               key={index}
+              color={"bg-hackathon-blue-100"}
```

Comment:
shouldn't be needed?

Reply:
if i remove it it says property color is missing up required in type

Reply:
it should be made optional then as a type

Reply:
should i do this inside Checkbox.jsx or Mentors.tsx